<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gxCubeDialog</class>
 <widget class="QDialog" name="gxCubeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btkCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Cube building</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Top surface</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Bottom surface</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Base grid surface</string>
        </property>
        <property name="buddy">
         <cstring>cbBaseGrid</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cbBaseGrid"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Sedimentation Model</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="cbSedimentationModel"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Number of Sublayers</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineNumberOfSublayers"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="dropWidgetTop">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="dropWidgetBottom">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>gxCubeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btkCancel</sender>
   <signal>clicked()</signal>
   <receiver>gxCubeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gxCubeDialog</sender>
   <signal>accepted()</signal>
   <receiver>gxCubeDialog</receiver>
   <slot>createCube()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>createCube()</slot>
 </slots>
</ui>
